helm/boot.sh

```diff
@@ -4,7 +4,7 @@ mkdir ~/.funcx/$1
 mkdir ~/.funcx/credentials
 cp /funcx/config/config.py ~/.funcx
 cp /funcx/$1/* ~/.funcx/$1
-cp /funcx/credentials/* ~/.funcx/credentials
+cp /funcx/credentials/storage.db ~/.funcx/
```

Comment:
Does this need to be `/funcx/storage.db`? I don't know how this file is getting here, but given that the file moved up and out of the `credentials/` subdir, I don't know if that impacts how this is generated.

Reply:
Confirmed because this dirname comes from the mountpoint used to get the secret into the pod